sdk/communication/Shared/src/CommunicationIdentifierSerializer.cs

```diff
@@ -46,29 +49,34 @@ private static CommunicationCloudEnvironment Deserialize(CommunicationCloudEnvir
         public static CommunicationIdentifierModel Serialize(CommunicationIdentifier identifier)
             => identifier switch
             {
-                CommunicationUserIdentifier u => new CommunicationIdentifierModel(CommunicationIdentifierKind.CommunicationUser)
-                {
-                    Id = u.Id
-                },
-                CallingApplicationIdentifier a => new CommunicationIdentifierModel(CommunicationIdentifierKind.CallingApplication)
+                CommunicationUserIdentifier u => new CommunicationIdentifierModel
                 {
-                    Id = a.Id
+                    CommunicationUser = new CommunicationUserIdentifierModel
```

Comment:
I was thinking shortly whether we should add `RawId = a.Id` but I agree that we shouldn't.